be/src/exec/spill/spiller.h

```diff
@@ -54,6 +54,11 @@ struct SpillProcessMetrics {
     RuntimeProfile::Counter* restore_rows = nullptr;
     RuntimeProfile::Counter* shuffle_timer = nullptr;
     RuntimeProfile::Counter* split_partition_timer = nullptr;
+
+    RuntimeProfile::Counter* flush_bytes = nullptr;
+    RuntimeProfile::Counter* restore_bytes = nullptr;
```

Comment:
must flush_bytes and restore_bytes be the same value?

Reply:
> must flush_bytes and restore_bytes be the same value?

no, for group by with limit, if spilling happens, agg block source will use sort-based aggregation, it may only need to restore a small part of data to return result, so restore_bytes may be smaller than flush_bytes